netlify.toml

```diff
@@ -9,7 +9,7 @@
     '''
     Content-Security-Policy = '''
       default-src 'self' https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://monitoring.lunie.io:9000 https://js.intercomcdn.com;
-      connect-src 'self' https://api-iam.intercom.io https://widget.intercom.io wss://production-api.lunie.io/graphql https://production-api.lunie.io https://monitoring.lunie.io:9000;
+      connect-src 'self' https://api-iam.intercom.io https://widget.intercom.io wss://production-api.lunie.io/graphql https://production-api.lunie.io wss://staging-api.lunie.io/graphql https://staging-api.lunie.io https://monitoring.lunie.io:9000;
       img-src 'self' data: https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://s3.amazonaws.com https://lunie.fra1.digitaloceanspaces.com;
       script-src 'self' 'unsafe-inline' 'unsafe-eval' https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://api-iam.intercom.io https://js.intercomcdn.com https://widget.intercom.io;
```

Comment:
I guess in the deployed build we should also not allow netlify.app

Reply:
I don't understand very well yet the Netlify architecture, that is why I left it there